enhancements/20230830-managed-dhcp.md

```diff
@@ -0,0 +1,318 @@
+# Managed DHCP
+
+## Summary
+
+We want a mechanism for VMs attached to VM networks to have network connectivity out of the box, and such funtionality is provided by Harvester itself, not external services.
```

Comment:
From the discussion with customer, if I am understanding correctly:

This feature is more an IP helper, it helps to allocate IP/gateway information to VMs and withdraw IPs quickly;  but the IP range allocation, the connectivity etc. are still sured by the infrastructure layer.
 
Harvester has no embedded router to route traffic between different L2 networks, the VxLan is also not supported yet. It is hard for Harvester to ensure the network connectivity independently now.
 
Due to those limitations, suggest to update the description of the functinality and motivation of this feature.

Reply:
Thanks for the suggestion. I've updated the paragraph; I hope that helps clarify the intention of the feature and the problem we want to solve.